config/base/derive/src/view.rs

```diff
@@ -46,45 +44,38 @@ mod gen {
-        // TODO: Remove `Default` from #[derive(..., Default, ...)] or #[derive(Default)] because we implement `Default` inside macro
-        // TODO: also add info with remove proxy
         ast.attrs
             .iter_mut()
             .filter(|attr| attr.path.is_ident("derive"))
             .for_each(|attr| {
                 let meta = attr
                     .parse_meta()
                     .expect("derive macro must be in one of the meta forms");
-                match meta {
-                    Meta::List(list) => {
-                        let items: Vec<syn::NestedMeta> = list
-                            .nested
-                            .into_iter()
-                            .filter(|nested| {
-                                if let NestedMeta::Meta(Meta::Path(path)) = nested {
-                                    // TODO: check here
-                                    if path.is_ident("Default")
-                                        || path.is_ident("Documented")
-                                        || path.is_ident("Proxy")
-                                    {
-                                        return false;
-                                    }
+                if let Meta::List(list) = meta {
+                    let items: Vec<syn::NestedMeta> = list
+                        .nested
+                        .into_iter()
+                        .filter(|nested| {
+                            if let NestedMeta::Meta(Meta::Path(path)) = nested {
+                                // remove derives that are needed on the `Configuration`
+                                // or `ConfigurationProxy`, but not needed on `ConfigruationView`
+                                if path.is_ident("LoadFromEnv")
+                                    || path.is_ident("Builder")
+                                    || path.is_ident("Proxy")
+                                {
+                                    return false;
                                 }
-                                true
-                            })
-                            .collect();
-                        *attr = syn::parse_quote!(
-                            #[derive(#(#items),*)]
-                        )
-                    }
-                    Meta::Path(path) if path.is_ident("Default") => {
-                        *attr = syn::parse_quote!(
-                            #[derive()]
-                        )
-                    }
-                    _ => {}
+                            }
+                            true
+                        })
+                        .collect();
+                    *attr = syn::parse_quote!(
+                        #[derive(#(#items),*)]
+                    )
                 }
             });
+        // TODO: Find a way to make this more ergonomic. As `View` struct
+        // are formed inside a proc macro, we have to remove unrelated attributes from `Configuration` here.
```

Comment:
Not a high priority problem. Create a good first issue and delegate to a FOSS contributor.

Reply:
Created #2773.